### PR TITLE
Fix bash completation

### PR DESCRIPTION
**Pull request by pshanoop**

### CODE REVIEW

**Comment by telemachus** [comment]
Thanks for doing this. I've been meaning to send this in too, but I wanted to look the
file over more.

I think [that line 62 also needs a small change](https://github.com/FiloSottile/passage/blob/5d687db06378032b17d7f8022a6e21cbc0ac591a/src/completion/pass.bash-completion#L62) too.

```bash
local prefix="${PASSWORD_STORE_DIR:-$HOME/.password-store/}"
# should be local prefix="${PASSWORD_STORE_DIR:-$HOME/.passage/store/}" like line 8, right?
```

**Comment by pshanoop** [comment]
> Thanks for doing this. I've been meaning to send this in too, but I wanted to look the file over more.
> 
> I think [that line 62 also needs a small change](https://github.com/FiloSottile/passage/blob/5d687db06378032b17d7f8022a6e21cbc0ac591a/src/completion/pass.bash-completion#L62) too.
> 
> ```shell
> local prefix="${PASSWORD_STORE_DIR:-$HOME/.password-store/}"
> # should be local prefix="${PASSAGE_DIR:-$HOME/.passage/store/}" like line 8, right?
> ```

Thanks, I will include that too.

File: src/completion/pass.bash-completion [hunk 1]
```diff
@@ -59,7 +59,7 @@ _pass_complete_entries () {
 }
 
 _pass_complete_folders () {
-	local prefix="${PASSWORD_STORE_DIR:-$HOME/.password-store/}"
+	local prefix="${PASSWORD_STORE_DIR:-$HOME/.passage/store/}"
```
Comment (telemachus) [inline src/completion/pass.bash-completion:62]: Sorry, but I think it needs one more tweak: `local prefix="${PASSAGE_DIR:-$HOME/.passage/store/}"`. I think I left in the `PASSWORD_STORE_DIR` in my first version of the comment. My mistake.

Comment (pshanoop) [reply src/completion/pass.bash-completion]: Updated.

**Pull request merged**